include/boost/detail/winapi/dll.hpp

```diff
@@ -1,7 +1,8 @@
 //  dll.hpp  --------------------------------------------------------------//
 
 //  Copyright 2010 Vicente J. Botet Escriba
-//  Copyright 2015 Andrey Semashev
```

Comment:
Please, don't remove or replace copyright lines but only add new ones. Even if the year is the same.